test/jest/victory-bar/victory-bar.spec.js

```diff
@@ -0,0 +1,257 @@
+/*eslint-disable max-nested-callbacks */
+
+import React from "react";
+import { render, fireEvent, screen } from "@testing-library/react";
+import { range } from "lodash";
+import { VictoryBar, Bar } from "victory-bar";
+import { svgExpectations, getShapeUtils } from "../../svg-test-helper";
+
+describe("components/victory-bar", () => {
+  describe("default component rendering", () => {
+    it("accepts user props", () => {
+      const { container } = render(
+        <VictoryBar data-testid="victory-bar" aria-label="Chart" />
+      );
+
+      const svgNode = container.querySelector("svg");
+      expect(svgNode.getAttribute("data-testid")).toEqual("victory-bar");
+      expect(svgNode.getAttribute("aria-label")).toEqual("Chart");
+    });
+
+    it("renders an svg with the correct width and height", () => {
+      const { container } = render(<VictoryBar />);
+      const svg = container.querySelector("svg");
+      expect(svg.getAttribute("style")).toContain("width: 100%; height: 100%");
+    });
+
+    it("renders an svg with the correct viewBox", () => {
+      const { container } = render(<VictoryBar />);
+      const svg = container.querySelector("svg");
+      const viewBoxValue = `0 0 ${450} ${300}`;
+      expect(svg.getAttribute("viewBox")).toEqual(viewBoxValue);
+    });
+
+    it("renders 4 bars", () => {
+      const { container } = render(<VictoryBar />);
+      const bars = container.querySelectorAll("path");
+      expect(bars.length).toEqual(4);
+    });
+
+    it("renders each bar as a rectangle", () => {
+      const { container } = render(<VictoryBar />);
+      const barCommandStrings = Array.from(
+        container.querySelectorAll("path")
+      ).map((bar) => bar.getAttribute("d"));
+      barCommandStrings.forEach(svgExpectations.expectIsRectangular);
```

Comment:
This is a personal preference, but as we're making these changes I would prefer not to continue to nest test expectations themselves inside helpers. Something like `expect(isRectangular(bar)).toBeTrue()` is a little more readable and re-usable in multiple test files IMO.